src/core/comm/mpi_comm.cc

```diff
@@ -69,10 +69,8 @@ MPINetwork::MPINetwork(int argc, char* argv[])
   int provided, init_flag = 0;
   CHECK_MPI(MPI_Initialized(&init_flag));
   if (!init_flag) {
-    log_coll.fatal(
-      "MPI has not been initialized, it should be initialized by "
-      "the networking backend.");
-    LEGATE_ABORT;
+    MPI_Init_thread(0, 0, MPI_THREAD_MULTIPLE, &provided);
+    self_mpi_init = true;
```

Comment:
```suggestion
    self_init_mpi = true;
```

Reply:
:exclamation: why did this compile?? (I did compile it locally)